options.cpp

```diff
@@ -168,6 +169,8 @@ namespace
     Expiry options:\n\
        -e|--expire-tiles [min_zoom-]max_zoom    Create a tile expiry list.\n\
        -o|--expire-output filename  Output filename for expired tiles list.\n\
+       -B|--expire-bbox-size Max size for a polygon to expire the whole polygon,\n\
+                             not just the boundary.\n\
```

Comment:
You forgot to adapt the documentation. While you change that, can you please also adapt the number of options in README.md and docs/usage.md.